Lib/test/test_email/test_email.py

```diff
@@ -5541,7 +5542,11 @@ def test_long_headers_flatten(self):
         result = fp.getvalue()
         self._signed_parts_eq(original, result)
 
-
+class TestHeaderRegistry(TestEmailBase):
+    # See issue 93010.
```

Comment:
Please use `gh-93010`.

Reply:
I have made the requested changes; please review again. Thank you!